server/game/cards/04-MM/MarkOfDis.js

```diff
@@ -0,0 +1,26 @@
+const Card = require('../../Card.js');
+
+class MarkOfDis extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            target: {
+                cardType: 'creature',
+                gameAction: ability.actions.sequential([
+                    ability.actions.dealDamage({ amount: 2 }),
+                    ability.actions.conditional((context) => ({
+                        condition: () => context.target.location === 'play area',
+                        trueGameAction: ability.actions.untilEndOfMyNextTurn({
+                            targetController: context.target.controller,
```

Comment:
This is wrong I think - targetController should be 'any', 'current' or 'opponent', shouldn't it?